.github/workflows/release.yml

```
@@ -140,3 +146,27 @@ jobs:
             poetry run semantic-release publish -v DEBUG
           fi
           echo "::set-output name=url::https://pypi.org/project/phylum/${{ env.phylum_release_version }}/"
+
+      - name: Build docker image
+        run: |
+          export PKG_SRC=dist/phylum-*.whl
+          export PKG_NAME=phylum-*.whl
+          docker build --tag phylumio/phylum-ci:latest --build-arg PKG_SRC --build-arg PKG_NAME .
+          docker tag phylumio/phylum-ci:latest phylumio/phylum-ci:${{ env.phylum_release_version }}
+
+      - name: Test docker image with pre-built distributions
+        run: |
+          docker run --rm phylumio/phylum-ci:latest "git --version"
+          docker run --rm phylumio/phylum-ci:latest "phylum-ci --version"
+          docker run --rm phylumio/phylum-ci:latest "phylum-ci --help"
+          docker run --rm phylumio/phylum-ci:latest "phylum-init --help"
+          docker run --rm phylumio/phylum-ci:latest "phylum --help"
+
+      - name: Push docker image
+        run: |
+          docker login --username ${{ secrets.DOCKER_HUB_USERNAME }} --password ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+          docker push --all-tags phylumio/phylum-ci
```

`--all-tags` seems dangerous... This line should only be pushing `:latest`, right?

---

It will push the two image tags made from the previous commands:

```sh
# This command makes the `latest` tag
docker build --tag phylumio/phylum-ci:latest --build-arg PKG_SRC --build-arg PKG_NAME .

# This command makes the tag from the phylum-ci release version, as a copy of `latest`
docker tag phylumio/phylum-ci:latest phylumio/phylum-ci:${{ env.phylum_release_version }}
```

Is the suggestion to change this to multiple `docker push` commands, one for each image tag specified explicitly?

---

Yes. I think that would be a safer alternative.

```suggestion
          docker push phylumio/phylum-ci:${{ env.phylum_release_version }}
          docker push phylumio/phylum-ci:latest
```

---

roger that...changed...plus some errors in the way the release workflow calls the docker image.